src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -106,7 +106,7 @@ class VolumeBar extends Slider {
    */
   updateARIAAttributes(event) {
     // Current value of volume bar as a percentage
-    const volume = (this.player_.volume() * 100).toFixed(2);
+    const volume = Math.round((this.player_.volume() * 100).toFixed(2));
```

Comment:
no need to the `.toFixed()`, that's what adds the two decimal places. Maybe just a `.toString()`